wg-charter.html

```diff
@@ -0,0 +1,387 @@
+          <p>The WebGPU specification should contain a section detailing any <a href='https://www.w3.org/TR/security-privacy-questionnaire/'>known security</a>, fingerprinting, and privacy implications, and suggested <a href='https://w3c.github.io/perf-security-privacy/'>mitigation strategies</a> for implementers, Web authors, and end users. The group should not publish a specification if acceptable mitigation strategies cannot be found.</p>
+          <p>The WebGPU specification should contain a section describing known impacts on accessibility to users with disabilities, ways the specification features address them, and recommendations for minimizing accessibility problems in implementation.</p>
+          <p>Normative specification changes are generally expected to have a corresponding set of tests, either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the proposed update.</p>
+        </div>
+
+        <h2>Out of Scope</h2>
+        <p>
+            The scope of work is restricted to the development of a programming
+            interface between the Web Platform and modern 3D graphics and
+            computation capabilities present on native system platforms. The work
+            will not define hardware features or algorithms, and the interface it
+            defines is not intended to be directly exposed by a GPU driver.
+        </p>
+
+      </section>
+
+
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <div id="normative">
+          <h3>
+            Normative
+          </h3>
+          <dl>
+          <dt>WebGPU API</dt>
+
+          <dd>
+              An API as described by the <a
+              href="https://www.w3.org/community/gpu/">GPU for the Web
+              Community Group</a> charter.
+              <p class="milestone"><b>Recommendation expected completion:</b> Q2 2020</p>
+          </dd>
+
+          <dt>WebGPU Test Suite</dt>
+
+          <dd>
+              A conformance test suite for the WebGPU API.
+              <p class="milestone"><b>Expected delivery:</b> Q2 2020</p>
+          </dd>
+
+          </dl>
+
+          <h3>
+            Optional Specifications
+          </h3>
+          <dl>
+          <dt>WebGPU Shading Language</dt>
+
+          <dd>
+              A Shading Language specification that defines the programmable
+              interface to the GPU for the Web graphics and computation
+              pipeline. Alternatively, the Working Group may recommend
+              an existing Shading Language.
+          </dd>
+          </dl>
+        </div>
+
+        <div id="wg-other-deliverables">
+            <h3>
+              Other deliverables
+            </h3>
+            <p>
+                The GPU for the Web Community group will produce deliverables to be
```

Comment:
This seems a bit open-ended and could be interpreted as allowing the WG to adopt any potentially normative **specification** deliverables into this group. I don't think that's the intent here, and so I would like to see the the "in particular" items that are expected to come from the GPU for the Web group called out a bit more explicitly and perhaps drop the "will produce deliverables to be used within this..." bit.